common/test/common/ModelOrResultTest.scala

```diff
@@ -15,7 +16,7 @@ import scala.concurrent.Future
 
 private object TestModel
 
-class ModelOrResultTest extends AnyFlatSpec with Matchers with WithTestExecutionContext {
+class ModelOrResultTest extends AnyFlatSpec with Matchers with WithTestExecutionContext with EitherValues {
```

Comment:
"ScalaTest's `EitherValues` trait provides an implicit conversion that adds `left.value` and `right.value` methods to `Either`, which will return the selected value of the `Either` if defined": https://www.scalatest.org/user_guide/using_EitherValues